### PR TITLE
add intake-stac

### PR DESCRIPTION
**Pull request by jhamman**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jhamman** [comment]
ping @ocefpaf and @matthewhanson to confirm you'd like to be maintainers here.

File: recipes/intake-stac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "intake-stac" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (jhamman) [inline recipes/intake-stac/meta.yaml]: @ocefpaf - again, I'm having trouble with this URL scheme. Ideas?

https://pypi.org/project/intake-stac/

Comment (ocefpaf) [reply recipes/intake-stac/meta.yaml]: PyPI mangles `-` and `_` in a very confusing way. In some places they are interchangeable, in other they are not.

**Comment by ocefpaf** [comment]
Failing with:

```
      File "c:\users\appveyor\appdata\local\temp\1\pip-req-build-qunff3\setup.py", line 7, in <module>
        import versioneer
    ImportError: No module named versioneer
```

That is happening b/c some files are missing the the source distribution. See https://github.com/pangeo-data/intake-stac/pull/21 for a fix.

File: recipes/intake-stac/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: d452cfee1ce041af12e2827f9f250ca18ce3c9bb6780b0920b90efdc2448a5fc
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
```
Comment (ocefpaf) [inline recipes/intake-stac/meta.yaml]: This can still be `noarch` if we constrain the python version.

**Comment by jhamman** [comment]
@ocefpaf - we should be good here now. Thanks for the help!

**Comment by matthewhanson** [comment]
Yes, I'll be maintainer @chrisburr @jhamman @ocefpaf

**Comment by ocefpaf** [comment]
> Looks good to me once @matthewhanson comments about if they want to be a maintainer

@jhamman if you want to merge right now we can remove @matthewhanson, merge, and re-add in the feedstock once we get a response.

**Pull request merged**